src/libdredd/src/mutation_replace_expr.cc

```diff
@@ -584,10 +584,10 @@ void MutationReplaceExpr::ReplaceExprWithFunctionCall(
   // cast.
   auto parents = ast_context.getParents<clang::Expr>(*expr_);
   // This the expression occurs in an initializer list and is subject to an
-  // explicit cast then it will have two parents -- the initializer list, and
-  // the implicit cast. (This is probably due to implicit casts being treated
-  // as invisible nodes in the AST.)
-  if (ast_context.getLangOpts().CPlusPlus && parents.size() == 2 &&
+  // explicit cast then it will have at least two parents -- the initializer
```

Comment:
In the above line please change "This the expression" to "This expression" (existing wording issue of mine that I noticed while reviewing this).

Reply:
Rephrased to
"The expression occurs in an initializer list and is then subject to an explicit cast will have at least two parents -- the initializer list, and the implicit cast. "